src/components/editTestCase/EditTestCase.tsx

```diff
@@ -292,13 +286,7 @@ const EditTestCase = () => {
       .then((tc: TestCase) => {
         setTestCase(_.cloneDeep(tc));
         setEditorVal(tc.json ? tc.json : "");
-        setCanEdit(
-          measure?.createdBy === userName ||
-            measure?.acls?.some(
-              (acl) =>
-                acl.userId === userName && acl.roles.indexOf("SHARED_WITH") >= 0
-            )
-        );
+        setCanEdit(canEdit);
```

Comment:
This seems unnecessary. Isn't it just setting the state variable "canEdit" to itself?